src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -220,28 +225,22 @@
   (let [user-contact @(re-frame/subscribe [:multiaccount/contact])
         contact-names @(re-frame/subscribe [:contacts/contact-two-names-by-identity pinned-by])]
     ;; We append empty spaces to the name as a workaround to make one-line and multi-line label components show correctly
-    (str "                   " (if (= pinned-by (user-contact :public-key)) (i18n/label :t/You) (first contact-names)))))
+    (str " " (if (= pinned-by (user-contact :public-key)) (i18n/label :t/You) (first contact-names)))))
```

Comment:
nit: as far as I can see all translation label keys are lower case expect this one. In my opinion we should adjust for consistency.

Reply:
I see the translation files have both `:t/you` and `:t/You`. So it seems intentional.